core/domain/recommendations_services.py

```diff
@@ -331,6 +336,14 @@ def set_recommendations(exp_id, new_recommendations):
 def get_exploration_recommendations(exp_id):
     """Gets a list of ids of at most 10 recommended explorations to play
     after completing the exploration keyed by exp_id.
+
+    Args:
+        exp_id: str. The ID of the exploration for which to get
+            the recommendations.
+
+    Returns:
+        list(str). List of recommended explorations IDs. Some of these IDs are
+        no longer valid/existing and need to be verified.
```

Comment:
need --> will need

That said -- all this seems a bit gross. We should make sure that the recommendations are valid, probably by doing a one-off job to delete models that aren't and handling on-the-fly deletion correctly going forward.

Reply:
I agree I think we should add a TODO to fix this (i think it is a bit out of the scope of wipeout, also it wasn't introduced by this PR), I checked that all the code that works with these models already expects this behaviour I only added this to the class/function documentation for other developers.

Reply:
I think the change is that we are now deleting these models as part of wipeout, so we should clean everything else up to make it consistent with that?

Reply:
I had some wrong assumptions about the time it would take to fix this. Actually it will be simple, we just need to run the `CronExplorationRecommendationsHandler` when the release is done.